### DISCUSSION TITLE
Exclude content from subfolders?

### DISCUSSION THREAD

**Discussion by iveptawebadmin** [estruyf/vscode-front-matter#172] [Q&A] [answered]
On my Hugo site, many of my pages like `about.md`, `contact.md`, etc. are in the root of my `content` directory.
Then I have subdirectories for subsections of the site like `blog`, `events`, etc. that have pages within them.

I would like the root content pages to be shown in a 'Pages' section, and then news posts to be in a 'News' section and so on.
Is there a way to set this up so that the 'Pages' folder does not look into any of the subdirectories within the `content` directory?

```
"frontMatter.content.pageFolders": [
    {
      "title": "Pages",
      "path": "[[workspace]]/content"
    },
    {
      "title": "News",
      "path": "[[workspace]]/content/news"
    }
  ],
```

Right now the 'Pages' folder has all the files from every subdirectory within `content`.

**Comment by estruyf** [comment, marked as answer]
This enhancement will be implemented in version `5.5.0`. Check its related issue: https://github.com/estruyf/vscode-front-matter/issues/174
